lib/consultation_reporter.rb

```diff
@@ -19,6 +20,24 @@ def all_consultations
     write_csv_from_scope(:all_consultations, scope)
   end
 
+  def response_percentages
+    scope = Consultation
+      .distinct('editions.id')
```

Comment:
Might be nice to use `.distinct(editions: :id)`. (I think that's possible in ActiveRecord...)